pkg/model/dev.go

```diff
@@ -823,3 +824,13 @@ func (s *Secret) GetKeyName() string {
 func (s *Secret) GetFileName() string {
 	return filepath.Base(s.RemotePath)
 }
+
+//ExpandEnv expands the environments supporting the notation "${var:-$DEFAULT}"
+func ExpandEnv(value string) string {
+	result, err := envsubst.String(value)
+	if err != nil {
+		log.Errorf("error expanding variables on '%s': %s", value, err.Error())
```

Comment:
Log.error logs to the UI. Instead of this return an error, and the caller can handle ir.

If this fails we shouldn't proceed, since we'd be having bad data.